pkg/interceptors/database_logging.go

```diff
@@ -32,11 +33,13 @@ func DatabaseLoggingUnaryServerInterceptor() grpc.UnaryServerInterceptor {
 			return nil, err
 		}
 
-		newDb := d.New()
-		newDb.LogMode(true)
-		newDb.SetLogger(sdklogger.NewGormLogger(l))
+		newDB := db.Session(&gorm.Session{
+			Logger:  sdklogger.NewGormLogger(l),
+			Context: ctx,
```

Comment:
@laynax After thinking more about this, I come to the conclusion that we don't need to pass context to a session in `interceptors` and `middlewares`. Usually, in `chassis` applications the context is passed down to the endpoint and that is up to the user to provide context, like in [document-search for an example](https://github.com/scribd/document-search/blob/cdcaa7c5009b413f48160211b01720ee994970fe/pkg/search/search.go#L196). wdyt?

Sorry for additional comment here 🤦

Reply:
already replied.

Reply:
@laynax I think this is a leftover, but the [Context](https://github.com/scribd/go-sdk/blob/cfa3d41cf97695c888bbf173774b7b4a50e63001/pkg/interceptors/database_logging.go#L38) is still there

Reply:
Ah sorry, it's fixed now.